enterprise/payara.common/nbproject/org-netbeans-modules-payara-common.sig

```diff
@@ -1803,7 +1805,7 @@ meth protected boolean getJdbcDriverDeploymentProperty()
 meth protected boolean getLoopbackProperty()
 meth protected boolean getPreserveSessionsProperty()
 meth protected boolean getStartDerbyProperty()
-meth protected void store(boolean,boolean,boolean,boolean,boolean,boolean,org.netbeans.modules.payara.common.PayaraInstance)
+meth protected void store(boolean,boolean,boolean,boolean,boolean,boolean,boolean,org.netbeans.modules.payara.common.PayaraInstance)
```

Comment:
This is exported API (fried-only) was it checked, that all friends are compatible?

Reply:
`org.netbeans.modules.payara.common.ui.InstancePanel#store` is part of `payara.common` module and not used by any other modules.